src/FakeItEasy/Core/TypeCatalogue.cs

```diff
@@ -161,10 +161,10 @@ private static void WarnFailedToGetSomeTypes(Assembly assembly, ReflectionTypeLo
                 int notLoadedCount = ex.Types.Count(t => t is null);
                 writer.Write($"{notLoadedCount} type(s) were not loaded for the following reasons:");
                 writer.WriteLine();
-                foreach (var loaderException in ex.LoaderExceptions)
+                foreach (var loaderException in ex.LoaderExceptions.Where(ex => ex is object))
                 {
                     writer.Write(" - ");
-                    writer.Write(loaderException.GetType());
+                    writer.Write(loaderException!.GetType());

```

Comment:
We now it's not null since we filtered out the nulls
(but we had a potential bug there before I added the filter)